src/providers/data_providers/IBusinessImageProvider.ts

```diff
@@ -0,0 +1,6 @@
+import { BusinessInfoInput } from '../../models/data_models/types';
+import { Image } from '../../models/data_models/image';
+
+export interface IBusinessImageProvider {
```

Comment:
```suggestion
export interface BusinessImageProvider {
```

Also consider renaming this file to `businessImageProvider.ts`